internal/cfgplugins/mpls.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cfgplugins
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/featureprofiles/internal/helpers"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+)
+
+// Configure static MPLS label binding using OC on device
+func NewStaticMPLSLabel(t *testing.T, dut *ondatra.DUTDevice, lspName string, incomingLabel uint32, nextHopIP string) {
+
+	d := &oc.Root{}
+	fptest.ConfigureDefaultNetworkInstance(t, dut)
+	mplsCfg := d.GetOrCreateNetworkInstance(deviations.DefaultNetworkInstance(dut)).GetOrCreateMpls()
+	staticMplsCfg := mplsCfg.GetOrCreateLsps().GetOrCreateStaticLsp(lspName)
+	staticMplsCfg.GetOrCreateEgress().SetIncomingLabel(oc.UnionUint32(incomingLabel))
+	staticMplsCfg.GetOrCreateEgress().SetNextHop(nextHopIP)
+	staticMplsCfg.GetOrCreateEgress().SetPushLabel(oc.Egress_PushLabel_IMPLICIT_NULL)
+
+	gnmi.Update(t, dut, gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).Mpls().Config(), mplsCfg)
+
+}
+
+// Configure static MPLS label binding (LBL1) using CLI with deviation, if OC is unsupported on device
+func DeviationStaticMplsLspOCUnsupported(t *testing.T, dut *ondatra.DUTDevice, incomingLabel uint32, intfName string, nextHopIP string, protocolType string) {
```

Comment:
The deviation should be moved into the NewStaticMPLSLabel function.   For an example of the style, see
https://github.com/openconfig/featureprofiles/blob/b48665dd0752dc1c76e2c9be8397b837e957ca6b/internal/cfgplugins/sflow.go#L78